force-app/main/default/triggers/PreventUpdateSubmitted.trigger

```diff
@@ -0,0 +1,7 @@
+trigger PreventUpdateSubmitted on Compensation__c (before update) {
```

Comment:
you just need a trigger per Object, so you'll have to put this and the other trigger in a single one. If you have more than one trigger per object, you are gonna have issues related to the salesforce's order of execution. try something like this: 

trigger PreventDeleteSubmitted on Compensation__c (before delete, before update ) {
	//Here you'll have to call a **trigger handler class** that should have a method per **context trigger variable** (isUpdate, isInsert).
}